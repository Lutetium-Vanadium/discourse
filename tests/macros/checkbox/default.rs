fn main() {
    inquisition::questions![Checkbox { default: todo!() }];
}
