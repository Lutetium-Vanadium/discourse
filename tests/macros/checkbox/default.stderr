error: option `default` does not exist for kind `checkbox`
 --> $DIR/default.rs:2:40
  |
2 |     inquisition::questions![checkbox { default: todo!() }];
  |                                        ^^^^^^^
